app/src/main/res/menu/navigation.xml

```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
No need to move from XML to code, you can add the Admin panel to XML and just change the visibility base on permissions.
`menuItem.isVisible = false`

Reply:
Really don't need to move from XML to code here, but I came across a specific situation where I needed to customize a search view programmatically (take a look on `Menu.kt` for the ChatRoom), so I find interesting declaring it here in the same way.

We can change it later if really needed.

Reply:
Still think it is a lot of code that doesn't need to be code... Even for the ChatRoom, the only thing that really needs to be code is `stylizeSearchView()` and the listeners...